core/state/statedb.go

```diff
@@ -73,6 +74,16 @@ func (m *mutation) isDelete() bool {
 	return m.typ == deletion
 }
 
+type arbFiltered int
+
+const (
+	unFiltered arbFiltered = iota
```

Comment:
Sorry to be so picky, but since unfiltered is a single word in English. I'd either use `unfiltered` or `notFiltered` here.